### PR TITLE
Allow None legend edge

### PR DESCRIPTION
**Pull request by mepland**
Do not draw the legend frame when `colors['legend_edge']` is `None`.

### CODE REVIEW

File: dtreeviz/trees.py [hunk 1]
```diff
@@ -214,7 +214,12 @@ def ctree_leaf_distributions(self,
             ax.set_ylabel("samples by class", fontsize=fontsize, fontname=fontname, color=colors['axis_label'])
             ax.grid(visible=grid)
             ax.legend([bar_container0, bar_container1],
-                      [f'class {self.shadow_tree.classes()[0]}', f'class {self.shadow_tree.classes()[1]}'])
+                      [f'class {self.shadow_tree.classes()[0]}', f'class {self.shadow_tree.classes()[1]}'],
```
Comment (parrt) [inline dtreeviz/trees.py:217]: Can we replace this with a call to add_classifier_legend()?

Comment (mepland) [reply dtreeviz/trees.py]: Done!

Comment (mepland) [reply dtreeviz/trees.py]: image

**Comment by parrt** [comment]
works!  Screenshot 2022-12-28 at 5 32 14 PM

**Pull request merged**